plugins/ideas/index.html

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html>
+<html style="background-color: transparent;">
```

Comment:
Does this get rid of that grey bar at the top on the submit initiative page?

Reply:
Yeah that's the plan. Trying to make it blend a little better,